src/distributions/distributions.jl

```diff
@@ -0,0 +1,58 @@
+## Series of variation of the MvNormal distribution, different methods need different parametrizations ##
+abstract type AbstractPosteriorMvNormal{T} <:
+              Distributions.ContinuousMultivariateDistribution end
```

Comment:
Why is this needed vs. `AbstractMvNormal` or whatever it's called?

Reply:
I was not aware of `AbstractMvNormal` until recently. This is still practical to have `AbstractPosteriorMvNormal` to define stuff like `mean(d::AbstractPosteriorMvNormal) = d.\mu`